packages/css/src/plugins/expand-shorthands/utils.ts

```diff
@@ -74,6 +83,9 @@ export const isWidth = (node: Node): boolean => {
  */
 export const getWidth = (node: Numeric | Word | Func): string => {
   if (node.type === 'numeric') {
+    if (node.value === '0' && node.unit === '') {
```

Comment:
I guess this will affect all unit based properties ( eg `height`, `width` not only `flex`).
However, unit should be irrelevant for those cases too

Reply:
hrm yeah, these utils are just used for flex but they could be useful for any future shorthands. I'll refactor this a bit to be a bit more readable